### DISCUSSION TITLE
need $NOW work on flows

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#14357] [Feature Requests]
Hi,team:

I thinking the `$NOW` is very beautiful for date calculate, but can not work on flows now.

**Comment by imiskolee** [comment]
based on https://github.com/directus/directus/discussions/14334, I have a date calculate samples:

1. CurrentTime: YYYY-MM-DD hh:mm:ss
```
{{ Eval( new Date().toISOString().replace('T',' ').substring(0,19) )}}
```
2. Current Date: YYYY-MM-DD
```
{{ Eval( new Date().toISOString().replace('T',' ').substring(0,10) )}}
```
3. + N days
 ```
{{ Eval( new Date(new Date(new Date().getTime() + 10 * 86400* 1000)).toISOString().replace('T',' ').substring(0,19) )}}
```
- N days
```
{{ Eval( new Date(new Date(new Date().getTime() - 10 * 86400* 1000)).toISOString().replace('T',' ').substring(0,19) )}}
```

Current Month State Date
```
{{ Eval( new Date(new Date(new Date().getFullYear(),new Date().getMonth(),1)).toISOString().replace('T',' ').substring(0,19) )}}
```

Next Month Start Date:
```
{{ Eval( new Date(new Date(new Date().getFullYear(),new Date().getMonth() + 1,1)).toISOString().replace('T',' ').substring(0,19) )}}
```
Prev Month Start Date:
```
{{ Eval( new Date(new Date(new Date().getFullYear(),new Date().getMonth() - 1,1)).toISOString().replace('T',' ').substring(0,19) )}}
```

It's work, but too complex.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
